src/authorization/jwt/MsJwtProvider.ts

```diff
@@ -18,7 +18,7 @@ export const authorizeJwt = (
   issuer: string,
   identity: ccfapp.JwtAuthnIdentity,
 ): ServiceResult<string> => {
-  const policy = JwtValidationPolicyMap.read(issuer);
+  const policy = JwtValidationPolicyMap.read(issuer) || {};
```

Comment:
Shouldn't we return "issuer is not registered" error rather than using empty policy?
I guess I'm missing something, but I feel like there would be no check if issuer doesn't exist in JwtValidationPolicyMap?

Reply:
good suggestion